docs/source/user_guide/streets_3857.csv

```diff
@@ -0,0 +1,1865 @@
+,u,v,key,osmid,oneway,lanes,name,highway,maxspeed,reversed,length,geometry,access,bridge,ref,width,tunnel,junction
```

Comment:
Similar to above -- trim this to only relevant columns.

Reply:
GitHub agrees: 
![image](https://github.com/rapidsai/cuspatial/assets/3943761/3591a4e4-f698-4c14-9df4-807ee2024fcb)
https://docs.github.com/en/repositories/working-with-files/using-files/working-with-non-code-files#rendering-csv-and-tsv-data